browser-extension/public/background.js

```diff
@@ -73,7 +73,9 @@ function checkDNTSetting() {
             "Do Not Track is disabled. Please enable it in your Chrome settings for enhanced privacy.",
           priority: 2,
         });
-        setTimeout(() => {}, 30000); // im giving them 30 seconds to turn it on
+        setTimeout(() => {
+          notify = false;
```

Comment:
@deandrebaker I filled it in, so the flow is if the setting is disabled, we will remind them every minute if they still haven't turned it on. 

I set notify to true in the beginning so it doesn't spam the notifs, but after the delay the flag will get reset

Reply:
Okay, sounds good